src/app/Shared/Redux/Configurations/DashboardConfigSlicer.tsx

```diff
@@ -65,6 +66,12 @@ export interface DashboardResizeConfigActionPayload {
   span: number;
 }
 
+export interface DashboardOrderConfigActionPayload {
+  idx: number;
+  order: number;
```

Comment:
Maybe name these 2 fields:  `previousOrder` and `nextOrder`? Also, need to update `_version` for this slicer.